README.md

```diff
@@ -62,24 +62,23 @@ Issues labeled [help wanted](https://github.com/photoprism/photoprism/labels/hel
 [easy](https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy) can be
 good (first) contributions. 
 
-Follow us on [Twitter][twitter] to receive regular project updates and discuss development 
-related topics. Don't be afraid to ask stupid questions.
+Follow us on [Twitter][twitter] to receive regular project updates and discuss development related topics. Don't be afraid to ask stupid questions.
 
 ## Funding ##
 
 This software is about freedom and privacy but not necessarily about free beer. We feel like it
-would be a mistake to state there will be no costs, because clearly we have huge expenses, your server hardware
+would be a mistake to state there will be no costs because clearly, we have huge expenses, your server hardware
```

Comment:
I think it makes sense the comma that was there was removed, but I'm not sure why a new one was added